docs/batfish.config.js

```diff
@@ -0,0 +1,98 @@
+const webpack = require('webpack');
+const mapboxAssembly = require('@mapbox/mbx-assembly');
```

Comment:
Do we have to keep this dependency? Can we hard code some of the setting here instead?